### ISSUE TITLE
Check for appropriate repository location

### ISSUE THREAD

**Issue by dsheets** [AltGr/Camelus#14]
Camelus should fail a PR check if the PR adds a package to the wrong location in the repository. See ocaml/opam-repository#8447.

**Comment by AltGr** [comment]
Yes, opam is actually _extremely_ tolerant, and accepts `packages/**/name.version/opam`.
We'll need to restrict this anyway when we enable repository signing.
